components/Portfolio/Insurance/Deposits/ActiveDeposits/index.tsx

```diff
@@ -52,9 +52,9 @@ const ActiveDeposits: FC<ADProps> = ({ parentHeight, pools }: ADProps) => {
                                 onMouseEnter={() => setShowButton(i)}
                                 onMouseLeave={() => setShowButton(-1)}
                             >
-                                <TableCell>{pool.market}</TableCell>
-                                <TableCell>{toPercent(pool.apy.toNumber())}</TableCell>
-                                <TableCell>{pool.userBalance.toNumber()}</TableCell>
+                                <TableCell>{pool?.market}</TableCell>
+                                <TableCell>{toPercent(pool?.apy?.toNumber())}</TableCell>
+                                <TableCell>{pool?.userBalance?.toNumber()}</TableCell>
```

Comment:
If this is mapping over the pools and you need these because of an error there is probably something deeper wrong with loading the pools

Reply:
Updated. This was since toNumber() couldn't be called on apy which could be undefined